data/json/items/armor.json

```diff
@@ -10170,5 +10170,124 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
-  }
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "bigpack",
+    "description": "Prized by pack rats, may lead to a bad back.",
+    "weight": 3539,
+    "volume": 85,
+    "price": 20000,
+    "material": [ "steel", "leather" ],
+    "symbol": "[",
+    "color": "black",
+    "covers": [ "TORSO" ],
+    "coverage": 95,
+    "encumbrance": 35,
+    "storage": 400,
+    "warmth": 5,
+    "material_thickness": 6,
+    "use_action": {
+    "type": "holster",
+    "max_volume": 20,
+    "min_volume": 5,
+    "multi": 2,
+    "skills": [ "smg", "shotgun", "rifle", "bow", "crossbow" ] },
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "backpack_hiking",
+    "type": "ARMOR",
+    "name": "hiking backpack",
+    "description": "A large sized hiking backpack with plenty of storage space.",
+    "weight": 2291,
+    "volume": 55,
+    "price": 10000,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 75,
+    "encumbrance": 25,
+    "storage": 220,
+    "warmth": 6,
+    "material_thickness": 6,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Sheath blade",
+      "holster_msg": "You sheath your %s",
+      "min_volume": 1,
+      "max_volume": 10,
+      "multi": 2,
+      "draw_cost": 3,
+      "flags": [ "SHEATH_KNIFE", "SHEATH_SWORD" ]
+    },
+    "flags": [ "BELTED", "WATERPROOF", "ONLY_ONE", "OVERSIZE" ]
+  },
+  {
+    "id": "petpack",
+    "type": "ARMOR",
+    "name": "petpack",
+    "description": "Before the Cataclysm this would allow your four-legged friend to see the world, now it's used to shield them from the world.",
+    "weight": 1316,
+    "volume": 40,
+    "price": 7900,
+    "material": [ "cotton", "plastic" ],
+    "symbol": "[",
+    "color": "blue",
+    "covers": [ "TORSO" ],
+    "coverage": 45,
+    "encumbrance": 12,
+    "storage": 25,
```

Comment:
```suggestion
```
You don't need the storage for the petpack to function correctly.

Reply:
Hmm I could lower it to 15 since the only pockets on it are the side pockets. Hopefully I can find out why travis is giving me trouble.

Reply:
I think your use_action if improperly formatted (compared to others in the same file). You can use the Linter [here](http://dev.narc.ro/cataclysm/format.html) to check the chunks of code you touched.

Reply:
Cool thanks Ill try that.